py/mockbuild/util.py

```diff
@@ -913,11 +914,11 @@ def setup_default_config_opts(unprivUid, version, pkgpythondir):
 
     # configurable commands executables
     config_opts['yum_command'] = '/usr/bin/yum'
-    if os.path.isfile('/usr/bin/yum-deprecated'):
-        config_opts['yum_command'] = '/usr/bin/yum-deprecated'
```

Comment:
It isn't removed, but moved to the package_manager module - that makes it easier to deal with the fact the outer chroot might have a different view of this